lib/styles/settings/_colors.scss

```diff
@@ -101,15 +103,15 @@ $color-fail: $color-outrageous-orange;
 $color-fail-supporting: $color-melon-orange;
 $color-fail-background: $color-ice-orange;
 
-$color-shadow: rgba($color-firefly-black, 0.35);
-$color-success-shadow: rgba($color-success, 0.7);
-$color-danger-shadow: rgba($color-danger, 0.7);
-$color-warning-shadow: rgba($color-warning, 0.7);
+$color-shadow: rgba(var(--raw-black-rgb), 0.3);
+$color-success-shadow: rgba(var(--raw-green-500-rgb), 0.7);
+$color-danger-shadow: rgba(var(--raw-red-500-rgb), 0.7);
+$color-warning-shadow: rgba(var(--raw-gold-500-rgb), 0.7);
 
 // Tokens - shadows
-$color-neutrals-shadow-weak: rgba($color-firefly-black, 0.06);
-$color-neutrals-shadow: rgba($color-firefly-black, 0.08);
-$color-neutrals-shadow-strong: rgba($color-firefly-black, 0.12);
+$color-neutrals-shadow-weak: $color-default-shadow-weak;
```

Comment:
are they still used, or can we remove them?

Reply:
In Platform on the branch `colors-migration-shadows-3a` they are not used, so I'm removing theme here.